python/dask_cudf/dask_cudf/io/json.py

```diff
@@ -6,4 +6,17 @@
 
 import cudf
 
-read_json = partial(dask.dataframe.read_json, engine=cudf.read_json)
+
+def read_json(path, engine="auto", **kwargs):
+    # Wrap `dd.read_json` with special engine handling
+
+    # TODO: Add optimized code path to leverage the
+    # `byte_range` argument in `cudf.read_json` for
+    # local storage (see `dask_cudf.read_csv`)
+    if isinstance(engine, str):
+        # Pass `str` engine argument to `cudf.read_json``
+        engine = partial(cudf.read_json, engine=engine)
+    elif not callable(engine):
+        raise ValueError(f"Unsupported engine option: {engine}")
+    # Pass `callable` engine argument to `dd.read_json`
+    return dask.dataframe.read_json(path, engine=engine, **kwargs)
```

Comment:
I think I perhaps don't understand the backend hookup properly. Why do we have to do this both here and in the `CudfBackendEntryPoint`?